modules/preparing-an-inital-cluster-deployment-for-mce-disconnected.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-an-inital-cluster-deployment-for-mce-disconnected_{context}"]
+
+= Preparing an initial cluster deployment for multicluster engine for Kubernetes(MCE) Operator while disconnected
+
+You can mirror the required {product-title} container images, the multicluster engine for Kubernetes(MCE) Operator, and Local Storage Operator (LSO) into your local mirror registry in a disconnected environment.
+Ensure that you note the local DNS and port of your mirror registry.
+
+[NOTE]
+====
+To mirror your {product-title} image repository to your mirror registry, you can use either the `oc adm release image`
+or `oc mirror` command. In this procedure, the `oc mirror` command is used as an example.
+====
+
+.Procedure
+
+. Create a `<assets_directory>` folder to place valid `install-config.yaml` and `agent-config.yaml` files. This directory is used to store all the assets.
+
+. To mirror the specific {product-title} image repository, the MCE Operator and Local Storage Operator(LSO), create a `ImageSetConfiguration.yaml` file with the following settings:
+
++
+.Example `ImageSetConfiguration.yaml`
+
+[source,yaml]
+----
+  kind: ImageSetConfiguration
+  apiVersion: mirror.openshift.io/v1alpha2
+  archiveSize: 4 <1>
+  storageConfig: <2>
+    imageURL: <your-local-registry-dns-name>:<your-local-registry-port>/mirror/oc-mirror-metadata <3>
+    skipTLS: true
+  mirror:
+    platform:
+      architectures:
+        - "amd64"
+      channels:
+        - name: stable-4.12 <4>
+          type: ocp
+    additionalImages:
+      - name: registry.redhat.io/ubi8/ubi:latest
+    operators:
+      - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.12 <5>
+        packages: <6>
+          - name: multicluster-engine <7>
+          - name: local-storage-operator <8>
+----
+
++
+NOTE: This file is required by the `oc mirror` command when mirroring content.
+
++
+<1> Add `archiveSize` to set the maximum size, in GiB, of each file within the image set.
+<2> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values, unless you are using the Technology Preview OCI feature.
+<3> Set the registry URL for the storage backend.
+<4> Set the channel to retrieve the {product-title} images from.
+<5> Set the Operator catalog to retrieve the {product-title} images from.
```

Comment:
```suggestion
<5> Set the Operator catalog that contains the {product-title} images you are installing.
```
Rewording suggestion to avoid ending sentence with preposition